share/completions/gradle.fish

```diff
@@ -37,3 +37,24 @@ complete -c gradle -l continuous -s t -d 'Enable continuous build'
 complete -c gradle -l no-search-upward -s u -d 'Don\'t search in parent folders for settings file'
 complete -c gradle -l version -s v -d 'Print version'
 complete -c gradle -l exclude-task -s x -x -d 'Specify task to be excluded from execution'
+
+# https://github.com/hanny24/gradle-fish/blob/master/gradle.load
+function __cache_or_get_gradle_completion
+  mkdir -p "/tmp/gradle_completion_cache_for_$USER"
+  set -l hashed_pwd (echo $PWD | md5sum | awk '{ print $1 }')
```

Comment:
This could use the `__funced_md5` function - that uses `string` and also supports systems with an `md5` command.

It's defined in https://github.com/fish-shell/fish-shell/blob/cb9f4e621fcaf83ba292292a9815d216715f2934/share/functions/funced.fish#L1-L12. To use it, you should extract it into its own file and rename it something more generic - `fish_md5` maybe?